[2024-02-19T21:50:43.662+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: quarta_dag.tsk1 manual__2024-02-19T21:50:31.611557+00:00 [queued]>
[2024-02-19T21:50:43.667+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: quarta_dag.tsk1 manual__2024-02-19T21:50:31.611557+00:00 [queued]>
[2024-02-19T21:50:43.667+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-02-19T21:50:43.667+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2024-02-19T21:50:43.667+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-02-19T21:50:43.675+0000] {taskinstance.py:1300} INFO - Executing <Task(BashOperator): tsk1> on 2024-02-19 21:50:31.611557+00:00
[2024-02-19T21:50:43.679+0000] {standard_task_runner.py:55} INFO - Started process 206 to run task
[2024-02-19T21:50:43.680+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'quarta_dag', 'tsk1', 'manual__2024-02-19T21:50:31.611557+00:00', '--job-id', '74', '--raw', '--subdir', 'DAGS_FOLDER/quartadag.py', '--cfg-path', '/tmp/tmpdtodjayx']
[2024-02-19T21:50:43.681+0000] {standard_task_runner.py:83} INFO - Job 74: Subtask tsk1
[2024-02-19T21:50:43.712+0000] {task_command.py:388} INFO - Running <TaskInstance: quarta_dag.tsk1 manual__2024-02-19T21:50:31.611557+00:00 [running]> on host b46627357673
[2024-02-19T21:50:43.751+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=quarta_dag
AIRFLOW_CTX_TASK_ID=tsk1
AIRFLOW_CTX_EXECUTION_DATE=2024-02-19T21:50:31.611557+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-02-19T21:50:31.611557+00:00
[2024-02-19T21:50:43.751+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-02-19T21:50:43.752+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'sleep 5']
[2024-02-19T21:50:43.758+0000] {subprocess.py:86} INFO - Output:
[2024-02-19T21:50:48.759+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-02-19T21:50:48.774+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=quarta_dag, task_id=tsk1, execution_date=20240219T215031, start_date=20240219T215043, end_date=20240219T215048
[2024-02-19T21:50:48.817+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2024-02-19T21:50:48.826+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
